### DISCUSSION TITLE
Cómo puedo invitar a un usuario no editor a una task?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#178] [2. Questions and Answers] [answered]
La pregunta es simple: Cómo puedo invitar usuarios a una task pero que sean seguidores o visualizadores?

Una de las formas que veo es mediante un update task y coloco los id de los usuarios. El problema con eso es que si debo invitar según por roles o cargo, mediante un selector, podría usar $CODE, pero el esquema no me quedaría bien claro.

Otra instancia era mediante por el pb addtaskeditor, que encuentro un bot muy potente y recurrente, pero sólo permite agregar usuarios a editar en una tasks, sería bueno "**modificar**" ese bot o crear otro que sea addtaskuser, que contenga todos los campos del bot indicado anterior + un campo indicando el tipo del usuario (editor, followers, visibility).

Espero que me puedan ayudar con esta consulta
Gracias :)

**Comment by rafaelwalter** [comment, marked as answer]
@NicolasPLG existe un `PBUpdateTask`, y tambien `NWRequest` que se puede ocupar para actualizar las tasks. Actualmente tenemos un update de la documentacion que se esta trabajando y en los proximos dias deberia estar disponible.

**Reply by acedward** [reply]
https://github.com/Cotalker/documentation/blob/26e1afb50c73875444a5c776bb7e55ba2ec3316b/docs/documentation/automation/bots/pbupdatetask.md

**Reply by NicolasPLG** [reply]
@rafaelwalter  gracias por la respuesta, si el updatetask hace eso, pero tengo que ver cómo hacer para que queden los usuarios mediante un rol en específico.
